lib/compress/index.js

```diff
@@ -5177,6 +5177,49 @@ merge(Compressor.prototype, {
             return true;
         }
 
+        function can_inject_args_values() {
+            var arg_fns_outer_refs = Object.create(null);
+            var value_walker = new TreeWalker(function(node) {
+                if (node instanceof AST_Scope) {
+                    var var_list = [];
+                    var var_outer_ref_map = Object.create(null);
+                    node.enclosed.forEach(function(def) {
+                        var_list.push(def.name);
+                        var_outer_ref_map[def.name] = true;
+                    });
+                    node.variables.each(function(def, name) {
+                        var_list.push(name);
+                        var_outer_ref_map[name] = false;
+                    });
+                    var_list.forEach(function(name) {
```

Comment:
I'm iterating over the set now, however, if I've read the contributing guidelines correctly, we can't use the fat arrow right? Since that is ES6 and the code is supposed to be ES5?

Reply:
Since recent commit we dropped out node 4.x support and now can use such things like Set, Map, arrow functions and other stuff supported by node 6.x and upper: https://node.green

Reply:
Should I replace all lambda's with arrow functions then? Or keep it consistent with the rest of the compress lib?

Reply:
I think it would be better to keep it consistent